OpenSID/OpenDK
# feat: Title Browser Beranda menggunakan nama kecamatannya

## Description
Add new feature based on this issue https://github.com/OpenSID/OpenDK/issues/143
- Set default browser title to nama_wilayah + nama_wilayah_kab
- Add new row to table das_profile
- Add ability to update the title from admin setting page
- Add the ability to update the title with the default browser title

## TODO
- [x] Set default browser title to `Beranda | Kecamatan Ile Boleng, Kab. Flores Timur`
- [x] Add setting page to update browser title
- [x] Add raw database to save the custom browser title
- [x] Update database seeder to insert new menu for dashboard setting

## How to test?
1. Run database migration.
```bash
php artisan database migrate
```
2. Then open home page, we should see the browser title like this `Kecamatan Ile Boleng, KAB. FLORES TIMUR`
http://localhost:8000

3. You can edit the browser title from this page, and then submit. The home page will appear with the new custom title.
http://localhost:8000/setting/dashboard

## Demo
https://www.loom.com/share/d0ad7ffe7fe248ffb52c6fed2a36ba23
